### PR TITLE
Issue #1553 feat : facilitator and learner pre-post requisites mapped

### PR DESCRIPTION
**Pull request by suvarnakale**

### CODE REVIEW

File: src/components/TopicDetails.tsx [hunk 1]
```diff
@@ -175,46 +177,200 @@ const TopicDetails: React.FC<TopicDetailsProps> = ({
...
+                          >
+                            {item?.name || subTopic.join(', ')}
+                          </Box>
+                        </Box>
+                      </a>
+                    </Grid>
+                  ))
+              ) : (
+                <Typography sx={{ m: 2 }}>
+                  {' '}
```
Comment (itsvick) [inline src/components/TopicDetails.tsx:292]: @suvarnakale there is new component for this, use same

File: src/components/SessionCardFooter.tsx [hunk 2]
```diff
@@ -52,12 +52,12 @@ const SessionCardFooter: React.FC<SessionCardFooterProps> = ({
     const fetchTopicSubtopic = async () => {
       try {
         const response = await getTargetedSolutions({
-          subject: 'English',
+          subject: 'Science',
           state: 'Maharashtra',
-          medium: 'Hindi',
+          medium: 'English',
```
Comment (itsvick) [inline src/components/SessionCardFooter.tsx:57]: @suvarnakale why its still hard coded?

Comment (suvarnakale) [reply src/components/SessionCardFooter.tsx:57]: Becoz for event I haven't integrated taxonomy yet

File: src/components/TopicDetails.tsx (hunk 1 above)
Comment (suvarnakale) [reply src/components/TopicDetails.tsx]: ok

**Pull request merged**